### ISSUE TITLE
About Window

### ISSUE THREAD

**Issue by smaragdus** [pyIDM/pyIDM#49]
This is a minor issue but as I encountered it I will report it.

When I go to `Settings` tab and hit `about` button it opens it right but the focus is still on the main window, screen:

![pyIDM 2020 2 22 - 2020-02-29 - error - 010](https://user-images.githubusercontent.com/9025757/75595232-3b9c9a80-5a94-11ea-8ed0-ace6634ea69f.png)

I think that when `about` is opened it should get the focus.

I also noticed that I can open as many instances of `about` as I want, screen:

![pyIDM 2020 2 22 - 2020-02-29 - error - 012](https://user-images.githubusercontent.com/9025757/75595402-eca33500-5a94-11ea-819b-a896f71057f1.png)

I think that the program should not allow to open multiple instances of `about` window- if an instance of `about` window is already opened trying to open another should just bring the focus to the already opened `about` window.

**Comment by pyIDM** [comment]
@smaragdus 
this is really a good note, I will work on it
Please don't hesitate to report anything, no matter it is big or small, bug or suggestion, everything matter
this is how this application will be improved

**Comment by smaragdus** [comment]
@pyIDM 

I discovered [pyIDM](https://github.com/pyIDM/pyIDM) yesterday at [VideoHelp](https://www.videohelp.com/software/pyIDM), decided to test it as it looked nice and it turned out to be a very promising video downloader, thanks for the good work. As the merging problem (#47) is now resolved I will do more tests. I may suggest a new feature but I will do it perhaps tomorrow, I am very tired right now.

Regards

**Comment by pyIDM** [comment]
Done

**Comment by smaragdus** [comment]
@pyIDM 

Thanks. I checked versions [2020.3.2](https://github.com/pyIDM/pyIDM/releases/tag/2020.3.2) and [2020.3.3](https://github.com/pyIDM/pyIDM/releases/tag/2020.3.3) and while now only a single instance of 'About' window can be opened the focus is still on the main window. Anyway, this is not a significant issue.

**Comment by smaragdus** [comment]
@pyIDM 

I can confirm that in version [2020.4.8](https://github.com/pyIDM/pyIDM/releases/tag/2020.4.8) 'About' window gets the focus, screen:

![pyIDM 2020 04 08 - 2020-04-10 - about - 002](https://user-images.githubusercontent.com/9025757/78986211-69fda300-7b33-11ea-9fa5-c368520d0c6a.png)

I have also noticed that 'About' button also got skinned, screen:

![pyIDM 2020 04 08 - 2020-04-10 - about - 004](https://user-images.githubusercontent.com/9025757/78986267-8f8aac80-7b33-11ea-852e-277913f17b38.png)

This is nice.

Off-topic- I am following [pyIDM](https://github.com/pyIDM/pyIDM) project since 2020-02-28 and I can say that within less than two months the program has improved incredibly. I am really impressed and I am convinced that you work on this project with dedication and passion!

**BIG THANK YOU!**

**Comment by smaragdus** [comment]
@pyIDM 

A further improvement of the 'About' window would be clickable links:

- **Clicking on URL link opens default web browser:**
- **Clicking on mail link opens default mail client;**

**screen:**

![pyIDM 2020 04 08 - 2020-04-10 - about - 005](https://user-images.githubusercontent.com/9025757/78987220-0e80e480-7b36-11ea-8a95-77052859425d.png)

This would be useful for quick access to project page via 'About' window.

**Comment by pyIDM** [comment]
@smaragdus 
Thanks a lot for your nice comments
clickable links is the right thing to do, will fix it in next release

one thing caught my attention:
in your screenshot i found segment size = 512.0 MB, it is too high value and you will get only one (worker/connection) while downloading files <= 512 MB, resulting in sometimes to very low speed.
this value should be 512 KB by default "less than one megabyte"

![image](https://user-images.githubusercontent.com/58998813/79009844-c62aec00-7b60-11ea-9d01-7b98b6a784ac.png)


my guess, you copied the last settings file from previous portable version which was easy to accidentally change combo box from KB to MB by mouse wheel

![image](https://user-images.githubusercontent.com/58998813/79010195-c081d600-7b61-11ea-8db0-70685e8ede6d.png)

**Comment by PySimpleGUI** [comment]
I see you've called `set_cursor` in your code, so you already know about the call, but maybe not for this use.  For clickable links, I've found this works well:

```python
window[text_key].set_cursor('hand2')
```

`hand2`  gives you this kind of cursor which seems right for links:

![image](https://user-images.githubusercontent.com/46163555/79015068-13479780-7b3a-11ea-9cd9-4fc5102b4efd.png)

**Comment by pyIDM** [comment]
@PySimpleGUI 
Nice hint, thanks a lot

**Comment by smaragdus** [comment]
@pyIDM 

Many thanks for noticing the bad parameter for segment size and for taking your time to show me screens with explanations and tips! I am pretty sure that you are right and I changed the parameter by mistake via the mouse wheel without noticing it. I corrected it.

I thought about opening a new issue about clickable links but I though it would be too much. So thank you for opening a separate issue (#72).

As you seem to be a perfectionist I won't hesitate to open new issues even for tiny glitches. I suppose that you don't mind that as it would result in better accountability.

Off-topic- once it was a hobby for me to test programs but as my time is limited now and I am spending most of my free time reading articles and books I test only a few programs from time to time. I can say that [pyIDM](https://github.com/pyIDM/pyIDM) caught my attention at first glance and I realized that it has huge potential and I was not mistaken. Keep the great work!

@PySimpleGUI 

I am not a programmer but I would like to thank you for creating [PySimpleGUI](https://github.com/PySimpleGUI/PySimpleGUI) which makes programs like [pyIDM](https://github.com/pyIDM/pyIDM) possible. I once tested [Python Easy Chess GUI](https://github.com/fsmosca/Python-Easy-Chess-GUI) and I like that it is portable and sports a clean GUI but I find it a bit rough and unfinished.
